main.go

```diff
@@ -36,6 +36,12 @@ func main() {
 	webhookListenerPort := os.Getenv("WEBHOOK_LISTENER_PORT")
 	workflowsDefTOMLFile := os.Getenv("WORKFLOWS_DEF_TOML_FILE")
 
+	if debug {
+		log.Println("Debug:", debug)
+		fmt.Printf("Loaded environment variables from %s\n", *envFile)
+		fmt.Printf("Using database file %s\n", dbFile)
```

Comment:
There are some differences between `log.Println()` vs `fmt.Println()` which I want to understand better and make a better call, but lets do that in future revisit of the entire logging.

Reply:
Oops, didn't notice it wasn't `fmt`, fixed.